### ISSUE TITLE
JSON : Arraw line are mixed on large json file 

### ISSUE THREAD

**Issue by Maxouwell** [plantuml/plantuml#1860]
**Describe the bug**
When displaying a json file with @startjson, the lines can become mixed and hard to follow .

**To Reproduce**
```
@startjson
{
 "1": "00000000-0000-0000-0000-000000000000",
 "1" : {"1": "00000000-0000-0000-0000-000000000000", "1" : { "1" : "1", "2": "2"}, "2" : { "1" : "1", "2": "2"},"1": "00000000-0000-0000-0000-000000000000"},
 "1" : {"1": "00000000-0000-0000-0000-000000000000", "1" : { "1" : "1", "2": "2"}, "2" : { "1" : "1", "2": "2"},"1": "00000000-0000-0000-0000-000000000000"},
 "1" : {"1": "00000000-0000-0000-0000-000000000000", "1" : { "1" : "1", "2": "2"}, "2" : { "1" : "1", "2": "2"},"1": "00000000-0000-0000-0000-000000000000"},
 "1" : {"1": "00000000-0000-0000-0000-000000000000", "1" : { "1" : "1", "2": "2"}, "2" : { "1" : "1", "2": "2"},"1": "00000000-0000-0000-0000-000000000000"},
 "1" : {"1": "00000000-0000-0000-0000-000000000000", "1" : { "1" : "1", "2": "2"}, "2" : { "1" : "1", "2": "2"},"1": "00000000-0000-0000-0000-000000000000"},
 "1" : {"1": "00000000-0000-0000-0000-000000000000", "1" : { "1" : "1", "2": "2"}, "2" : { "1" : "1", "2": "2"},"1": "00000000-0000-0000-0000-000000000000"},
 "1" : {"1": "00000000-0000-0000-0000-000000000000", "1" : { "1" : "1", "2": "2"}, "2" : { "1" : "1", "2": "2"},"1": "00000000-0000-0000-0000-000000000000"},
 "1" : {"1": "00000000-0000-0000-0000-000000000000", "1" : { "1" : "1", "2": "2"}, "2" : { "1" : "1", "2": "2"},"1": "00000000-0000-0000-0000-000000000000"},
 "1": "00000000-0000-0000-0000-000000000000"
}
@endjson
```

result in 
![image](https://github.com/user-attachments/assets/fe6a87a1-2018-4993-b989-c74860703d04)
https://www.plantuml.com/plantuml/png/SoWkIImgoIhEp-Egvb9GCbIoKb0oW09T3081b7J0gXMi5Af9qGJJ0w6194Z8ICaAHDI2c7XaIR2dbggk6tNTgEj6NJVgEXAKSzLoEQJcfO3ALG00

**Expected behavior**
Easy to follow arrow lines

**Screenshots**
I have an internal case where the behavior is worse, but I'm not sure I can share the source :
![image](https://github.com/user-attachments/assets/6496edff-25de-4b51-b466-4087775e1914)

**Comment by The-Lum** [comment]
See similar issue/request here:
- https://forum.plantuml.net/13090/arrows-not-correct-with-json-or-yaml-big-data-smetana

**Comment by antonianemi** [comment]
Maybe using a Straigth lines could help, this is my veri first time here so i can try to contribute.
